diktat-rules/src/test/resources/test/paragraph3/long_line/LongBinaryExpressionExpected.kt

```diff
@@ -1,35 +1,39 @@
 package test.paragraph3.long_line
 
 fun foo() {
-    val veryLongExpression = Methoooooooooooooooood() +
- 12345
+    val veryLongExpression =
```

Comment:
Why did you change already existing logic about line breaking?

Reply:
Ok, in this case probably yes, but there are also another situations, see for example my next comment